### DISCUSSION TITLE
What is `dressing.nvim` used for in `lazyvim`?

### DISCUSSION THREAD

**Discussion by hopezh** [LazyVim/LazyVim#1352] [Q&A] [answered]
1. The first quesiton: 
May I ask **_how [dressing.nvim](https://github.com/stevearc/dressing.nvim) is utilized in lazyvim_**, e.g. which interface is "enhanced", or changed, by dressing.nvim? (I checked the [config of dressing.nvim in lazyvim](https://www.lazyvim.org/plugins/ui#dressingnvim), and I didn't understand what it is doing, unfortunately...)

2. The second question:
Or, put it in another way, **_what will be affected in lazyvim if `dressing.nvim` is disabled?_**

I'm asking the second question because I need to disable it to avoid some conflicts with another plugin [quarto-nvim](https://github.com/quarto-dev/quarto-nvim) I'm using, as suggested by the maintainer of quarto.nvim [@jmbuhr](https://github.com/quarto-dev/quarto-nvim/discussions/82#discussioncomment-6763099:~:text=%23diff%2Dae0a74392a85a01dcf65a8b24a487b6035159ab5a5801df01320e45706ecd32c-,jmbuhr,-15%20hours%20ago) over [here](https://github.com/quarto-dev/quarto-nvim/discussions/82#discussioncomment-6763094).

Thanks.

**Comment by dpetka2001** [comment, marked as answer]
I can't give you an answer based on the code that it's written. However, I disabled it and the only thing I observed was that the `vim.ui.input` box that appears for user input (for example when you do a LSP rename of a symbol) appears at the top middle of the screen instead of near the cursor when used with `dressing.nvim` enabled. I didn't find any breakages in my short test with `dressing` disabled, so I guess you would be safe disabling it and see how it goes.
